projects/app.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var proj = [];
+function Projects(myProject) {
+  Object.keys(myProject).forEach(x => this[x] = myProject[x]);
+}
+
+Projects.prototype.insertion = function() {
+  var $newProject = $('.template').first().clone();
+
+  $newProject.find('a').attr('href', this['url']);
+  $newProject.find('img').attr('src', this['imgSrc']);
+  $newProject.find('h3').text(this['title']);
+  $newProject.find('p').text(this['body']);
+  return $newProject;
+};
+
+projectObjects.forEach(item => proj.push(new Projects(item)));
+
+proj.forEach(item => $('main').append(item.insertion()));
+
+$('.template').first().remove();
```

Comment:
It is better practice to remove the class of template from the new projects rather than completely removing the original template itself.